lab_1/main.py

```diff
@@ -1,12 +1,18 @@
-mat = input('''Введите текст, который нужно обработать или имя файла, который нужно открыть (с форматом)''')
-if mat[-4:0] == ".txt":
-    f = open(mat,'r')
-    text = f.read()
-else:
-    text = mat
-def calculate_frequences(text) -> dict:
+import random
+
+
+def read_from_file():
+    f = open(r'C:\Users\admin\Downloads\2018-2-level-labs\Beatles - Yesterday.txt', 'r')
+    c = f.read()
+    return c
+
+
+text = read_from_file()
```

Comment:
@kazadvornov set any sample text here, reading from file in this function breaks all tests